openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1281,6 +1289,10 @@ def create_openmm_system(self, topology, **kwargs):
         """
         return_topology = kwargs.pop("return_topology", False)
 
+        if "Electrostatics" not in self.registered_parameter_handlers:
+            # Could do this later, to not raise when systems don't have any charges?
```

Comment:
There could be a more elegant way to handle this, I guess, if we're concerned about force fields that have no electrostatics. Relates to if that is a requirement in the spec.